### PR TITLE
Add CPU database variable

### PR DESCRIPTION
**Pull request by felixtheflex**
Paramitising the CPU element of the Postgres Arm Template SKU.

### CODE REVIEW

**Comment by felixtheflex** [comment]
[Scale up and out API app service](https://trello.com/c/Xce9e8q7/213-scale-up-and-out-api-app-service)

File: templates/postgresql-server.json [hunk 1]
```diff
@@ -8,16 +8,20 @@
...
-      "type": "object",
-      "defaultValue": {
-        "name": "GP_Gen5_2",
-        "tier": "GeneralPurpose",
-        "size": 5120,
-        "family": "Gen5",
-        "capacity": 2
-      }
-    },
+    "cpu" : {
```
Comment (vigneshmsft) [inline templates/postgresql-server.json]:
```suggestion
    "dbServerCpuCores" : {
```

File: templates/postgresql-server.json [hunk 2]
```diff
@@ -8,16 +8,20 @@
...
+      "type": "int",
+      "defaultValue": 2
+  },
+  "postgresSku": {
+    "type": "object",
+    "defaultValue": {
+      "name": "GP_Gen5_2",
```
Comment (vigneshmsft) [inline templates/postgresql-server.json]:
```suggestion
      "name": "[concat('GP_Gen5_', parameters('dbServerCpuCores'))]",
```
or just

```suggestion
      "name": "GP_Gen5",
```

File: templates/postgresql-server.json [hunk 3]
```diff
@@ -8,16 +8,20 @@
...
+      "tier": "GeneralPurpose",
+      "size": 5120,
+      "family": "Gen5",
+      "capacity": "[parameters('cpu')]"
```
Comment (vigneshmsft) [inline templates/postgresql-server.json]:
```suggestion
      "capacity": "[parameters('dbServerCpuCores')]"
```

File: templates/postgresql-server.json (hunk 2 above)
Comment (felixtheflex) [reply templates/postgresql-server.json]: see this:- https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-postgresql-server-database-using-arm-template?tabs=azure-portal

Comment (vigneshmsft) [reply templates/postgresql-server.json]: ah okie..thought 2 referrred to the number of cores. ignore this one. thanks.

Comment (vigneshmsft) [reply templates/postgresql-server.json]: actually https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-azure-cli#create-an-azure-database-for-postgresql-server

**Review by vigneshmsft — APPROVED** [review]

**Review by saliceti — APPROVED** [review]

**Pull request merged**